challenges/cyclical_linked_list.py

```diff
@@ -0,0 +1,21 @@
+def has_cycle(head):
+    if is_terminated_soon(head):
+        return False
+    else:
+        return is_cyclical(head, head.next.next)
```

Comment:
That won't work. is_terminated_soon being True means has_cycle needs to short circuit to false.